features/shared/checkout_previous_branch_after_success/deleted_current_branch/previous_branch_deleted.feature

```diff
@@ -0,0 +1,11 @@
+Feature: when the previous and current branches are deleted during a Git Town command, the main branch becomes the new previous branch
```

Comment:
Feature: deleting current and previous branch makes the main branch the new previous branch